docs/observability/application-components.md

```diff
@@ -271,13 +267,53 @@ Explore is an out-of-the-box Sumo Logic navigation tool that provides an intuiti
 The **Application Components - Database System Overview** dashboard appears in the window on the right. The dashboard provides an at-a-glance view of the different database engines present in the database component and insights on errors occurring across different engines.
 1. And then keep on going down the hierarchy to specific entities for each of the components to view dashboards at more granular levels.
 
+### Dashboard Examples
+
```

Comment:
@JKashyap96 , Is the Application Components Solution related to the APM Dashboards? I've inserted your screenshots here for the time being.